go-chaos/internal/chaos-experiments/camunda-cloud/production-s/follower-terminate/experiment.json

```diff
@@ -15,7 +15,19 @@
                 "tolerance": 0,
                 "provider": {
                     "type": "process",
-                    "path": "verify-readiness.sh",
+                    "path": "zbchaos",
+                    "arguments": ["verify", "readiness"],
+                    "timeout": 900
```

Comment:
:question: Not related to this PR. I was wondering if we can configure this somewhere so we don't have to repeat it in every action and every experiment. We should only specify if it is different from the default "900".

Reply:
Actually It think we already have a default of 15 min here https://github.com/zeebe-io/zeebe-chaos/blob/main/go-chaos/worker/chaos_worker.go#L57

Reply:
:+1: May be that is enough.